jib-core/src/main/java/com/google/cloud/tools/jib/builder/steps/BuildImageStep.java

```diff
@@ -119,23 +119,19 @@
       imageBuilder.addLabels(baseImage.getLabels());
 
       // Add history elements for non-empty layers that don't have one yet
-      int sizeDifference =
-          baseImageLayers.size() + buildAndCacheApplicationLayerSteps.size() - nonEmptyLayerCount;
-      for (int count = 0; count < sizeDifference; count++) {
-        imageBuilder.addHistory(
-            new HistoryItem(
-                containerConfiguration == null
-                    ? Instant.EPOCH.toString()
-                    : containerConfiguration.getCreationTime().toString(),
-                "Jib",
-                "jib",
-                null));
+      String layerCreationTime =
+          containerConfiguration == null
+              ? Instant.EPOCH.toString()
+              : containerConfiguration.getCreationTime().toString();
+      for (int count = 0; count < baseImageLayers.size() - nonEmptyLayerCount; count++) {
+        imageBuilder.addHistory(new HistoryEntry(layerCreationTime, "Jib", "jib", null));
```

Comment:
But I don't think this should be "Jib"/"jib" — maybe author = _"(unknown)"_ and createdBy= _"(unavailable)"_?

Reply:
I can change them, but right now createdBy is just a placeholder. After this I think I'm going to add a way to pass in something like "jib-gradle-plugin" or "jib-maven-plugin". Maybe I should do the same for author?

@coollog WDYT

Reply:
I think Brian means for the base image layers that don't have history to have `author` and `createdBy` be `(unknown)`, though I think leaving them empty would be fine too.

Reply:
Oh ok, makes sense.